proxy/client/go.mod

```diff
@@ -36,9 +36,9 @@ require (
 	github.com/subosito/gotenv v1.4.1 // indirect
 	github.com/ugorji/go/codec v1.2.7 // indirect
 	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e // indirect
-	golang.org/x/net v0.0.0-20221014081412-f15817d10f9b // indirect
```

Comment:
@fm100 Can we bump that? GitHub shows security vulnerability here

Reply:
Yes, we can. I think it's not going to break anything, and even if it breaks thing, It's still in development and not being used anyone, and it doesn't harm anything anyway 😄

Reply:
@mobuchowski I still don't have a permission to merge. Can you merge this?